book/website/community-handbook/coworking/coworking-dropins.md

```diff
@@ -0,0 +1,90 @@
+# Drop-in Sessions
+
+Drop-in sessions are an opportunity for a team or group of people to host regular opportunities for other people within the community or organisation to ask questions or start a conversation on a topic of their interest.
```

Comment:
```suggestion
Drop-in sessions are an informal collaboration format where a team or group of people host regular opportunities for other people within the community or organisation to ask questions or start a conversation on a topic of their interest.
```

double use of opportunities otherwise